carla/recourse_methods/catalog/dice/model.py

```diff
@@ -76,8 +82,11 @@ def get_counterfactuals(self, factuals):
         )
 
         cf_ex_list = dice_exp.cf_examples_list
-        cf = pd.concat([cf_ex.final_cfs_df for cf_ex in cf_ex_list], ignore_index=True)
-
+        cfs = pd.concat([cf_ex.final_cfs_df for cf_ex in cf_ex_list], ignore_index=True)
```

Comment:
Can we prefix dataframes with `df_` in this project? (i have seen this convention quite a lot). That makes it easier for the reader to understand what are operations on the dataframe and what not

Reply:
Sounds good 👍